services/QuillLMS/spec/models/user_spec.rb

```diff
@@ -96,6 +96,10 @@
   #it { should validate_uniqueness_of(:username).on(:create) }
 
   it { should validate_presence_of(:username).on(:create) }
+  it { should validate_length_of(:username).is_at_most(255) }
```

Comment:
@cissyxyu here too :-) 
Using the constant means we can change the value later without breaking specs.

Reply:
Whoops, makes sense! I added it in the spec too.